mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -37,6 +38,8 @@
 // remove this eventually
 ConVar sv_slope_fix("sv_slope_fix", "1");
 ConVar sv_ramp_fix("sv_ramp_fix", "1");
+ConVar sv_nojump_slope_boosts("sv_nojump_slope_boosts", "1", 0,
+    "Keep original engine behavior which makes going up slopes without holding jump usually result in extra speed.");
```

Comment:
As per Aux's request, you may remove this and treat the code that uses it as if it were hardcoded to 0.